docs/docs-content/getting-started/additional-capabilities.md

```diff
@@ -23,34 +23,56 @@ Palette provides comprehensive solutions to meet different business needs. Learn
+when the connection to the management plane is lost. You can manage Edge clusters locally on-site through the local UI,
+or centrally through the Palette management plane. Palette Edge is able to meet your needs, regardless of the network
+topology your deployments face. Check out the [Palette Edge](../clusters/edge/edge.md) page to learn more about Edge and
+its features.
 
 ## Self-Hosted Palette
 
-Palette is available as a fully self-hosted platform offering. You can install an instance of the Palette management
-plane in your data center or public cloud provider to manage Kubernetes clusters.
+Palette is available as a fully self-hosted platform offering that you can leverage to manage all your Kubernetes
+clusters. You can install an instance of the Palette management plane in your data center or public cloud provider using
+different hosting models.
 
-This option gives you maximum control over the platform, as you are responsible for the day-to-day management.
-Additionally, it enables deployments in air-gapped environments where access to the public cloud is not possible. Read
-more on the [Self-Hosted Palette](../enterprise-version/enterprise-version.md) page.
+This option allows you to meet compliance and legal requirements once you have full control over your workloads,
+determine who has access to them, and ensure that no sensitive data is exposed. You can also upgrade the platform at
+your own pace and configure system settings through the
+[System Console](../enterprise-version/system-management/system-management.md#system-console). Additionally, self-hosted
+Palette enables deployments in air-gapped environments where access to the public cloud is not possible. Explore more on
+the [Self-Hosted Palette](../enterprise-version/enterprise-version.md) page.
 
 ## Palette VerteX
 
-Palette VerteX offers a simple, flexible, and secure way for government and regulated industries to manage Kubernetes
-workloads containing sensitive and classified information. It is available as a self-hosted platform offering that you
-can install in your data center or public cloud provider.
+Palette VerteX offers a simple, flexible, and secure way for government and regulated industries to deploy and manage
+Kubernetes workloads containing sensitive and classified information. It is available as a self-hosted platform offering
+that you can install in various environments. These include virtualized and bare metal data centers (like
+[VMware vSphere](https://www.vmware.com/products/vsphere.html) and [Nutanix](https://www.nutanix.com/)), clouds
+(including [AWS](https://aws.amazon.com/govcloud-us/) and
+[Azure](https://azure.microsoft.com/en-ca/explore/global-infrastructure/government) government clouds), and edge
+locations (including air-gapped setups), which makes VerteX also appropriate for addressing challenges like intermittent
+connectivity or low bandwidth.
```

Comment:
@caroldelwing I feel like we are mixing two concepts here. The hosting model of VerteX, and what clusters you can deploy through VerteX.

Reply:
Yes, you're right :sweat_smile: I've separated the concepts and placed them in different paragraphs.